contracts/core/RebalancingSetToken.sol

```diff
@@ -241,8 +256,36 @@ contract RebalancingSetToken is
         // Must be in Rebalance state to call settlement
         require(rebalanceState == State.Rebalance);
 
+        // Make sure all currentSets have been rebalanced
+        require(remainingCurrentSets == 0);
+
+        // Create ICore object
+        ICore core = ICore(ISetFactory(factory).core());
+
+        // Issue nextSet and deposit in Vault
```

Comment:
This step doesn't deposit into the vault does it?

Reply:
+1